code/modules/clothing/head/misc.dm

```diff
@@ -285,7 +285,7 @@
 
 /obj/item/clothing/head/crown/fancy
 	name = "magnificent crown"
-	desc = "A crown worn by only the highest emperors of the land."
+	desc = "A crown worn by only the highest emperors of the <s>land</s> space."

```

Comment:
cross out the `the` as well

Reply:
This is my pirates of the pancreas

The joke works perfectly as is and I can't seem to get anyone to understand why it is good the way it is

Reply:
Basically it's supposed to be kind of an intentional mis-grammar in order to give the impression of space being almost a parallel to ancient medieval societies (which is not what you'd expect in space)